api/src/test/java/org/pmiops/workbench/api/AuthDomainControllerTest.java

```diff
@@ -28,6 +28,7 @@
 import org.pmiops.workbench.firecloud.FireCloudService;
 import org.pmiops.workbench.firecloud.model.FirecloudManagedGroupWithMembers;
 import org.pmiops.workbench.google.DirectoryService;
+import org.pmiops.workbench.mail.MailService;
```

Comment:
From here down, it's strictly adding required beans/mocks because these tests reference UserService ... and a little gratuitous alphabetization